index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>garden-vue</title>
+    <link rel="stylesheet" href="/static/normalize.css">
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
```

Comment:
Since this is *technically* your implementation I can't say no... But please, please don't use `jquery` unless there is a very good reason for it.